Essentials/src/main/resources/messages.properties

```
@@ -985,14 +985,17 @@ replyLastRecipientDisabledFor=\u00a76Replying to last message recipient \u00a7cd
 replyLastRecipientEnabled=\u00a76Replying to last message recipient \u00a7cenabled\u00a76.
 replyLastRecipientEnabledFor=\u00a76Replying to last message recipient \u00a7cenabled \u00a76for \u00a7c{0}\u00a76.
 requestAccepted=\u00a76Teleport request accepted.
+requestAcceptedAll=\u00a76Accepted \u00a7c{0} \u00a76pending teleport request(s).
```

Keeping the style the same as the original message.
```suggestion
requestAcceptedAll=\u00a76Teleport requests accepted from \u00a7c{0} \u00a76 players.
```